normandy/classifier/templates/admin/classifier_preview.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'admin/base_site.html' %}
+{% load static %}
+
+{% block extrastyle %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static 'admin/css/forms.css' %}" />
+  <link rel="stylesheet" type="text/css" href="{% static 'admin/css/classifier_preview.css' %}" />
+  {{ form.media.css }}
+{% endblock %}
+
+{% block extrahead %}
+  {{ block.super }}
+  <script type="text/javascript" src="{% static 'admin/js/core.js' %}"></script>
+  <script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
+  <script type="text/javascript" src="{% static 'admin/js/jquery.init.js' %}"></script>
+  <script type="text/javascript" src="{% url 'admin:jsi18n' %}"></script>
+  {{ form.media.js }}
+{% endblock %}
+
+{% block content %}
+  <form method="get">
+    {{ form.non_field_errors }}
+    <div class="client-properties">
+      <div class="field">
+        {{ form.locale.errors }}
+        {{ form.locale.label_tag }}
+        {{ form.locale }}
+      </div>
+      <div class="field">
```

Comment:
It can only be in a loop if we generate the list of fields in the view due to the limitations of Django templates. The best I think we can do is write an [inclusion tag](https://docs.djangoproject.com/en/1.9/howto/custom-template-tags/#inclusion-tags) and list them out explicitly, still.